nrf_security/include/mbedtls/platform.h

```diff
@@ -401,6 +401,10 @@ int mbedtls_platform_setup( mbedtls_platform_context *ctx );
  */
 void mbedtls_platform_teardown( mbedtls_platform_context *ctx );
 
+#ifdef CONFIG_MBEDTLS_DEBUG
+void zephyr_mbedtls_debug(void *ctx, int level, const char *file, int line, const char *str);
```

Comment:
Since you already have a patch to zephyr I don't think we need this change.
see comment: https://github.com/nrfconnect/sdk-zephyr/pull/1087#pullrequestreview-1332255733